identity-apps-core/apps/recovery-portal/src/main/webapp/password-recovery.jsp

```diff
@@ -408,7 +411,7 @@
                         <input type="hidden" name="channel" value=""/>
                         <input type="hidden" name="sp" value="<%=sp %>"/>
                         <input type="hidden" name="spId" value="<%=spId %>"/>
-                        <input type="hidden" name="urlQuery" value="<%=request.getQueryString() %>"/>
+                        <input type="hidden" name="urlQuery" value="<%=urlQuery %>"/>
```

Comment:
```suggestion
                        <input type="hidden" name="urlQuery" value="<%= urlQuery %>"/>
```

Reply:
lets remove the spaced around the vars.

Reply:
Its in this pattern in other places also..